titan-server/src/schema.rs

```diff
@@ -222,6 +222,21 @@ table! {
     }
 }
 
+table! {
+    user_event_excuses (id) {
+          id -> Integer,
+          user_id -> Integer,
```

Comment:
This is inconsistent with the SQL migration. In the migration, it's named `static12_id` but in the Rust definition it's being called `user_id`. I think it should be `legacy_player_id` in-line with our Titan user parlance.

Reply:
The `static12_id` field was meant to be replaced with `user_id` in an earlier refactor. Do we want to keep the legacy id with each excuse?

Reply:
I'm alright with replacing the `static12_id` with `user_id` and using the Titan `users` table's `id` field to join on it. You just need to make this consistent with the migration.